docs/backup_and_restore.rst

```diff
@@ -334,6 +338,74 @@ Migrating Using a V2+V3 or V3-Only Backup
 #. :doc:`Test the new instance <test_the_installation>` to verify that the
    web interfaces are available and the servers can be reached via SSH.
 
+.. _repair_admin_usbs:
+
+Repair Additional Admin Workstations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I would suggest a standard scrub of the below for italicizing glossary terms (I don't have strong attachments to the convention, but while we have it ..).

Reply:
sounds good. I find all the italics and capitalization of names really hard to read, personally, so when it says *Admin Workstation* and *Transfer Device* everywhere I find it confusing. I've changed to "LUKS-encrypted USB" here (fc894a2)